### PR TITLE
[#2800][#2801] fix(license): fix incorrect LICENSE information of Kafka source code

### PR DESCRIPTION
**Pull request by mchades**
### What changes were proposed in this pull request?

fix incorrect LICENSE information of Kafka source code

### Why are the changes needed?

Fix: #2800 #2801 

### Does this PR introduce _any_ user-facing change?

no

### How was this patch tested?

no need

### CODE REVIEW

**Comment by mchades** [comment]
Could you please help review it, thanks! @justinmclean

File: NOTICE
Comment (justinmclean) [inline NOTICE:1]: These changes are not correct. Why are you adding license information to the NOTICE file? And why are you adding things are are not include in teh source release?

**Comment by justinmclean** [comment]
You also need to update LICENSE.bin

File: NOTICE
Comment (justinmclean) [reply NOTICE:1]: Ah I see what has happened here, their NOTICE file is incorrectly formed. I would ignore all of it and just leave our NOTICE file as is.

Comment (mchades) [reply NOTICE:1]: yeah, I just copied their NOTICE file here, then should I only add Zookeeper here?

> Apache Zookeeper
> Copyright 2008-2023 The Apache Software Foundation

**Comment by mchades** [comment]
> You also need to update LICENSE.bin

Could you provide more information on what should be added to the LICENSE.bin file? Thx @justinmclean

File: NOTICE
Comment (justinmclean) [reply NOTICE:1]: I would add nothing, as I think everything is irrelevant. You would only add the Zookeeper bit if the code we copied came from Zookeeper.

**Comment by justinmclean** [comment]
Same changes as LICENSE

File: NOTICE
Comment (mchades) [reply NOTICE:1]: ok

**Comment by mchades** [comment]
> Same changes as LICENSE

But `Apache Kafka` and `Apache ZooKeeper` have already in LICENSE.bin

**Comment by justinmclean** [comment]
For LICENSE.bin just add under the Apaxchee 2.0 section: 
Confluent Kafka Streams Examples

**Pull request merged**